openssl/src/x509/extension.rs

```diff
@@ -428,6 +433,306 @@ impl AuthorityKeyIdentifier {
     }
 }
 
+/// A constructor for the `X509` AS number extension.
+#[cfg(ossl110)]
+#[cfg(not(OPENSSL_NO_RFC3779))]
+pub struct SbgpAsIdentifier(SbgpAsIdentifierOrInherit);
+
+#[cfg(ossl110)]
+#[cfg(not(OPENSSL_NO_RFC3779))]
+enum SbgpAsIdentifierOrInherit {
+    Inherit,
+    List(Vec<(u32, u32)>),
+}
+
+#[cfg(ossl110)]
+#[cfg(not(OPENSSL_NO_RFC3779))]
+impl Default for SbgpAsIdentifier {
+    fn default() -> SbgpAsIdentifier {
+        SbgpAsIdentifier::new()
+    }
+}
+
+#[cfg(ossl110)]
+#[cfg(not(OPENSSL_NO_RFC3779))]
+impl SbgpAsIdentifier {
+    /// Construct a new `SbgpAsIdentifier` extension.
+    pub fn new() -> SbgpAsIdentifier {
+        Self(SbgpAsIdentifierOrInherit::List(Vec::new()))
+    }
+
+    /// Sets the `inherit` flag to `true`.
+    pub fn add_inherit(&mut self) -> &mut SbgpAsIdentifier {
+        if let SbgpAsIdentifierOrInherit::List(ref l) = self.0 {
+            if !l.is_empty() {
+                panic!("cannot set extension to inherit, list allready contains elements");
```

Comment:
```suggestion
                panic!("cannot set extension to inherit, list already contains elements");
```

Similar typos are throughout the panic strings

Reply:
No problem,  I will fix all these